intermediate_html_css/intermediate_css_concepts/frameworks_and_preprocessors.md

```diff
@@ -64,3 +57,10 @@ This section contains questions for you to check your understanding of this less
 * [What are some disadvantages of using a CSS framework?](https://www.smashingmagazine.com/2018/11/css-frameworks-css-grid/)
 * [What are some advantages of using a CSS preprocessor?](https://medium.com/@LiliOuakninFelsen/what-is-a-css-pre-processor-why-should-you-use-one-ac3d4cd4b2c2)
 * [What are some disadvantages of using a CSS preprocessor?](https://adamsilver.io/blog/the-disadvantages-of-css-preprocessors/)
+
+### Additional Resources
```

Comment:
```suggestion
### Additional Resources

```

Reply:
Small thing to edit: all headings should have [a newline before and after it](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#newlines)

Thanks for making this PR